transform/app/services/s3/load.py

```diff
@@ -0,0 +1,57 @@
+import gzip
+import json
+import logging
+from io import BytesIO
+from typing import Dict, List
+
+from app.services.s3.connect import connect_to_s3
+from app.services.s3.utils import (
+    extract_bucket_and_directory,
+    list_files_in_zip,
+    process_json_content,
+    read_json_from_zip,
+)
+from app.settings import settings
+
+logger = logging.getLogger(__name__)
+
+
+def load_file_from_s3(file_path: str) -> List[Dict]:
+    """
+    Loads the file from S3 and returns its content as JSON object.
+
+    Supports .json and .json.gz formats. Each line in the file is treated
+    as a separate JSON document.
+    """
+    data = []
+    try:
+        s3_client = connect_to_s3(
+            settings.S3_ACCESS_KEY, settings.S3_SECRET_KEY, str(settings.S3_ENDPOINT)
+        )
+        bucket_name, key = extract_bucket_and_directory(file_path)
```

Comment:
If you write about bucket_name and key, this will be harder to extract because key is needed to use the `get_object' method.

Example key:
`test/software/part-00000.json`.

and it's use in the loop:
`s3_object = s3_client.get_object(Bucket=bucket_name, Key=key)`